asv_bench/benchmarks/io/csv.py

```diff
@@ -285,6 +285,15 @@ def time_read_uint64_na_values(self):
         )
 
 
+class ReadUint8Integers(StringIORewind):
```

Comment:
Why StringIORewind and not BaseIO?

Reply:
It is analogous to [`ReadUint64Integers`](https://github.com/pandas-dev/pandas/blob/d92103db9723cb74b728d8719265630fb06b0501/asv_bench/benchmarks/io/csv.py#L267). I am only interested in the parsing performance of uint8 so I do not need a real file (and the teardown of `BaseIO`) like most of the existing `ReadCSV...` asvs.